homeassistant/components/lyric/climate.py

```diff
@@ -190,7 +190,10 @@ def target_temperature(self) -> float | None:
         """Return the temperature we try to reach."""
         device = self.device
         if not device.hasDualSetpointStatus:
-            return device.changeableValues.heatSetpoint
+            if LYRIC_HVAC_MODE_COOL.lower() in self.hvac_mode:
```

Comment:
Only the standard hvac modes are allowed as hvac_mode. Please use the climate integration constants.

https://developers.home-assistant.io/docs/core/entity/climate#hvac-modes